monkey/infection_monkey/exploit/__init__.py

```diff
@@ -76,6 +76,16 @@ def add_executed_cmd(self, cmd):
         powershell = True if "powershell" in cmd.lower() else False
         self.exploit_info['executed_cmds'].append({'cmd': cmd, 'powershell': powershell})
 
+    def _try_lock(self, create_file_fn, path):
```

Comment:
There is no point in having this kind of function... I think the easiest solution would be to remove these methods from HostExploiter and remove the nesting of method definitions in `_create_lock_file` and `_remove_lock_file`. Do that and we'll merge :)